ppr-ui/src/components/parties/PartySearch.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+  <v-container fluid no-gutters class="white pa-0">
+    <v-row class="px-6" align="center">
+      <v-col cols="6">
+        <v-text-field
+          filled
+          label="Secured Party Code or Name"
+          id="txt-code"
+          :hide-details="true"
+          persistent-hint
+        />
+      </v-col>
+      <v-col cols="6">
+        or
+        <a id="add-party" @click="goToAddSecuredParty"
+          >Add a Secured Party that doesn't have a code</a
+        >
+      </v-col>
+    </v-row>
+    <v-row class="px-6" align="center">
+      <v-col cols="auto">
+        <v-checkbox
+          id="add-registering-party"
+          class="reg-checkbox pa-0 ma-0"
+          @click="addRegisteringParty"
+          :hide-details="true"
+          >
+        </v-checkbox>
+      </v-col>
+      <v-col>
+        Include the registering party as a secured party
+      </v-col>
+    </v-row>
+  </v-container>
+</template>
+
+<script lang="ts">
+import {
+  defineComponent,
+  reactive,
+  toRefs
+} from '@vue/composition-api'
+
+export default defineComponent({
+  emits: ['showSecuredPartyAdd'],
+  setup (props, context) {
+    const localState = reactive({})
```

Comment:
can localState be removed or are you expecting to add to this later?

Reply:
This is just the structure, the actual search is another ticket, so probably leave it in :)